ProcessLib/HT/CreateHTProcess.cpp

```diff
@@ -83,6 +68,13 @@ std::unique_ptr<Process> createHTProcess(
     DBUG("Use \'%s\' as fluid_reference_density parameter.",
          fluid_reference_density.name.c_str());
 
+    // Parameter for the density of the solid.
+    auto& density_solid = findParameter<double>(
+        config,
+        //! \ogs_file_param_special{prj__processes__process__HT__density_solid}
+        "density_solid", parameters, 1);
+    DBUG("Use \'%s\' as density_solid parameter.", density_solid.name.c_str());
+
```

Comment:
It is for slightly compressible fluid, e.g. water under high pressure. You can use 1 for biot_constant because the coupling of mechanical process can be neglected in the current status. Here is a reference about  the slightly compressible fluid (see eq. (5)).
http://onlinelibrary.wiley.com/doi/10.1029/JB090iB08p06817/pdf

Or you can keep the current implementation and add some words in documentation that fluid compressibility is not considered so far.